docs/sources/setup/upgrade/_index.md

```diff
@@ -36,13 +36,46 @@ The output is incredibly verbose as it shows the entire internal config struct u
 
 ## Main / Unreleased
 
+## 3.0.0
+
+{{% admonition type="note" %}}
+If you have questions about upgrading to Loki 3.0, please join us at https://slack.grafana.com/ in the channel `#loki-3`
+
+Or leave a comment on this [Github Issue](https://github.com/grafana/loki/issues/12506)
+{{% /admonition %}}
+
+Loki 3.0 is a major version increase and comes with several breaking changes.
+
+Here is the shortlist of things we think most people may encounter:
+
+  * Structured metadata is enabled by default and requires `tsdb` and `v13` schema or Loki won't start. Refer to [Structured Metadata, Open Telemetry, Schemas and Indexes](#structured-metadata-open-telemetry-schemas-and-indexes).
+  * The `shared_store` config is removed. Refer to [Removed `shared_store` and `shared_store_key_prefix` from shipper configuration](#removed-shared_store-and-shared_store_key_prefix-from-shipper-configuration).
+  * Loki now enforces a max line size of 256KB by default (you can disable this or increase this but this is how Grafana Labs runs Loki). Refer to [Changes to default configure values](#changes-to-default-configuration-values-in-30).
+  * Loki now enforces a max label limit of 15 labels per series, down from 30. Extra labels inflate the size of the index and reduce performance, you should almost never need more than 15 labels. Refer to [Changes to default configure values](#changes-to-default-configuration-values-in-30).
+  * There are many metric name changes. Refer to [Distributor metric changes](#distributor-metric-changes), [Embedded cache metric changes](#embedded-cache-metric-changes), and [Metrics namespace](#metrics-namespace).
+
+If you would like to see if your existing configuration will work with Loki 3.0:
+1. In an empty directory on your computer, copy you configuration into a file named `loki-config.yaml`.
+  * Run this command from that directory: 
+```bash
+docker run --rm -t -v "${PWD}":/config grafana/loki:3.0.0 -config.file=/config/loki-config.yaml -verify-config=true`
+```
+
+**Note:** if you introduce a new schema_config entry it may cause additional validation errors
+
+**Tip:** if you configure `path_prefix` in the `common` config section this can help save a lot of configuration. See the [Common Config Docs](https://grafana.com/docs/loki/latest/configure/#common)
```

Comment:
```suggestion
{{< admonition type="tip" >}}
If you configure `path_prefix` in the `common` config section this can help save a lot of configuration. Refer to the [Common Config Docs](https://grafana.com/docs/loki/latest/configure/#common).
{{< /admonition >}}

```